src/spiegel.js

```diff
@@ -77,6 +77,10 @@ class Spiegel {
       case 'uninstall':
         await this.uninstall()
         break
+
+      default:
+        log.fatal('unknown type: ' + this._type + ' v' + pkg.version)
```

Comment:
I think the idea is a good one, but instead of using `log.fatal()`, I'd recommend: `throw new Error('unknown type: ' + this._type)`